pkgs/desktops/mate/mate-tweak/default.nix

```
@@ -0,0 +1,82 @@
+{ stdenv
+, fetchFromGitHub
+, python3Packages
+, intltool
+, mate
+, libnotify
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+, glib
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mate-tweak";
+  version = "20.04.0";
+
+  src = fetchFromGitHub {
+    owner = "ubuntu-mate";
+    repo = pname;
+    rev = version;
+    sha256 = "14cvh5hf61s0z75skrpz6r0z27jd01z100v2cqih2ck0pxv0483v";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    intltool
+    python3Packages.distutils_extra
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    libnotify
+    glib
+    mate.mate-applets
+    mate.mate-panel
+    mate.marco
+    mate.libmatekbd
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    distro
+    pygobject3
+    psutil
+    setproctitle
+  ];
+
+  strictDeps = false;
+
+  dontWrapGApps = true;
+
+  postPatch = ''
+    # mate-tweak hardcodes absolute paths everywhere. Nuke from orbit.
+    find . -type f -exec sed -i \
+      -e s,/usr/lib/mate-tweak,$out/lib/mate-tweak,g \
+      {} +
+  '';
+
+  postInstall = ''
+    mv -v $out/${python3Packages.python.sitePackages}/nix/store/*/lib/* $out/lib
+    mv -v $out/${python3Packages.python.sitePackages}/nix/store/*/share/* $out/share
+    rm -r $out/${python3Packages.python.sitePackages}/nix
+  '';
+
+  # Arguments to be passed to `makeWrapper`, only used by buildPython*
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  postFixup = ''
+    sed -i "s|usr|run/current-system/sw|g" $out/bin/.mate-tweak-wrapped
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tweak tool for the MATE Desktop";
+    homepage = "https://github.com/ubuntu-mate/${pname}";
+    changelog = "https://github.com/ubuntu-mate/${pname}/releases/tag/${version}";
+    license = [ licenses.gpl2 ];
```

```suggestion
    license = [ licenses.gpl2Plus ];
```

https://github.com/ubuntu-mate/mate-tweak/blob/2ccffffdff2b2f831cd05f787e37c0f297c1422f/util/mate-tweak-helper#L9